components/Calendar/Calendar.jsx

```diff
@@ -1,10 +1,166 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
-
-// eslint-disable-next-line no-unused-vars
-export default (props) => (
-  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-    <Text>Calendar Screen</Text>
-    <Text>Replace these Text tags with your component</Text>
-  </View>
-);
+import { View, FlatList } from 'react-native';
+import { Calendar } from 'react-native-calendars';
+import styled from 'styled-components';
+import firebase from 'firebase';
+import { useDispatch, useSelector } from 'react-redux';
+import CalendarWorkoutCard from './CalendarWorkoutCard';
+import actions from '../../actions';
+
+const days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July',
+  'August', 'September', 'October', 'November', 'December'];
+
+const DayTitle = styled.Text`
+  font-family: 'Montserrat_500Medium';
+  font-size: 16px;
+  text-align: left;
+  margin-top: 25px;
+  margin-bottom: 15px;
+`;
+
+const NoWorkoutText = styled(DayTitle)`
+  font-size: 24px;
+`;
+
+function formatDate(date) {
+  return `${days[date.getDay()]}, ${months[date.getMonth()]} ${date.getDate() + 1}`;
+}
+
+function setDates(dates) {
```

Comment:
Can we rename this function to something more descriptive. Maybe markCalendarDates or something like that